src/main.cpp

```diff
@@ -1271,8 +1271,8 @@ std::list<libzerocoin::CoinDenomination> ZerocoinSpendListFromBlock(const CBlock
 
 bool CheckZerocoinMint(const uint256& txHash, const CTxOut& txout, CValidationState& state, bool fCheckOnly)
 {
-    if(!fCheckOnly && GetAdjustedTime() < GetSporkValue(SPORK_17_ENABLE_ZEROCOIN))
-        return state.DoS(100, error("CheckZerocoinMint(): Zerocoin transactions are not allowed yet"));
+    if(!fCheckOnly && chainActive.Height() >= Params().Zerocoin_StartHeight())
```

Comment:
You changed the operator here to reject all zerocoin transactions that have ever occurred.